src/components/worksFullPage/workCard/workCard.tsx

```diff
@@ -0,0 +1,30 @@
+import Image from "next/image";
+import cn from "classnames";
+
+import { IWorkCard } from "types/common/ComponentsProps";
+
+import style from "./workCard.module.scss";
+
+export const WorkCard = ({ cardSize, classname }: IWorkCard) => {
+	const workWrapperClasses = cn({
+		[style.workWrapper]: true,
```

Comment:
cn(style.workWrapper, { ...

if workWrapper is always true, better write it outside cn object